test/api/core_test.dart

```diff
@@ -161,7 +161,7 @@ void main() {
           ..which(condition));
     }
 
-    final zulipLocalizations = lookupZulipLocalizations(ZulipLocalizations.supportedLocales.first);
+    final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
```

Comment:
nit in commit message:

    store [nfc]: Refactor remaining inconsistent localization use in test

Prefix should be "test [nfc]:" — this commit isn't related to the store, but it is in tests. Then the "in test" at the end isn't needed, either.

(Alternatively it could be "api test [nfc]:" to indicate which area's tests it's about. But this particular change is really about the tests in general, and it just happens that there's only one example of this pattern.)